packages/edit-post/src/components/preferences-modal/index.js

```diff
@@ -102,18 +102,18 @@ export default function EditPostPreferencesModal() {
 							) }
 						>
 							<EnableFeature
-								featureName="reducedUI"
+								featureName="focusMode"
 								help={ __(
-									'Compacts options and outlines in the toolbar.'
+									'Highlights the current block and fades other content.'
 								) }
-								label={ __( 'Reduce the interface' ) }
+								label={ __( 'Spotlight mode' ) }
 							/>
 							<EnableFeature
-								featureName="focusMode"
+								featureName="showIconLabels"
+								label={ __( 'Show button text labels' ) }
 								help={ __(
-									'Highlights the current block and fades other content.'
+									'Show text instead of icons on buttons'
```

Comment:
If you'll permit me the tiniest of tiny nits: can we add a full-stop to the end of this line while we're here? It looks like this is the only one that doesn't have a full-stop (if so, we'd want to update it in the `edit-site` preferences, too).

<img src="https://user-images.githubusercontent.com/14988353/167751417-47bb169e-4484-4223-b7e8-d3a1f2b656db.png" width="550" />